code/go/0chain.net/conductor/config/node.go

```
@@ -111,10 +111,10 @@ func (n *Node) Stop() (err error) {
 		return fmt.Errorf("command %v: kill: %v", n.Name, err)
 	}
 	if stdin, ok := startCmd.Stdin.(*os.File); ok {
-		stdin.Close() // ignore error
+		stdin.Close() //nolint // ignore error
```

Isn't this an example of where we want to deliberately ignore an error?